bokehjs/src/lib/models/mappers/eqhist_color_mapper.ts

```diff
@@ -0,0 +1,63 @@
+import {ScanningColorMapper} from "./binned_color_mapper"
+import {Arrayable} from "core/types"
+import {min, max, bin_counts, interp} from "core/util/arrayable"
+import {linspace, cumsum, range} from "core/util/array"
+import * as p from "core/properties"
+
+export namespace EqHistColorMapper {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = ScanningColorMapper.Props & {
+      bins: p.Property<number>
+    }
+}
+
+export interface EqHistColorMapper extends EqHistColorMapper.Attrs {
```

Comment:
I think we should be explicit here too, esp since these are most likely not going ot be used by average users. `EqualHistogramColorMapper`